spec/model/taxon_spec.rb

```diff
@@ -8,13 +8,42 @@
     before(:each) do
       example_schema_string = GovukContentSchemaTestHelpers::Examples.new.get('taxon', 'taxon_with_child_taxons')
       @example_schema = JSON.parse(example_schema_string)
+      @topic_taxon = Taxon.new("education")
     end
 
     it "should return a content item from the content store" do
       allow(Services.content_store).to receive(:content_item).with("/education").and_return(@example_schema)
 
-      topic_taxon = Taxon.new("education")
-      expect(topic_taxon.title).to eq("Taxon with children")
+      expect(@topic_taxon.title).to eq("Taxon with children")
+    end
+
+    it "should return associated child taxons" do
+      allow(Services.content_store).to receive(:content_item).with("/education").and_return(@example_schema)
+
+      child_taxons = @topic_taxon.child_taxons
+
+      expect(child_taxons.length).to eq(1)
+      expect(child_taxons.first["title"]).to eq("Further and higher education, skills and vocational training")
+    end
+  end
+
+  describe "#children?" do
+    before(:each) do
+      @topic_taxon = Taxon.new("education")
+    end
+
+    it "should indicate that taxon has child taxons" do
+      example_schema_with_children = JSON.parse(GovukContentSchemaTestHelpers::Examples.new.get('taxon', 'taxon_with_child_taxons'))
```

Comment:
You could move this line and the equivalent in the test below to the `before` block.

Reply:
@leenagupte I think they're separate because one test needs taxons with children and one needs taxons without?